packages/client/.prettierrc

```
@@ -0,0 +1,7 @@
+{
+    "bracketSpacing": true,
+    "semi": false,
```

```suggestion
    "semi": true,
```

To be consistent with server rules.